nixos/modules/security/sudo.nix

```diff
@@ -171,12 +171,6 @@ in
 
   config = mkIf cfg.enable {
 
-    security.sudo.extraRules = [
-      { groups = [ "wheel" ];
```

Comment:
I think a `[ (mkBefore { groups = ... }) ]` might work

Reply:
I guess it should be `mkBefore [ ... ]`, that is, `mkBefore` for the whole list. At least when I played around with these a bit, I needed to use it like that - thanks to an advise at IRC from @jD91mZM2.